### ISSUE TITLE
explain whether to aim for speed or accuracy or both

### ISSUE THREAD

**Issue by annecori** [mrc-ide/typochallenge#65]
I wasn’t sure if I was meant to be aiming for speed or accuracy or both. I then saw I got rewards for all of these, but only after going to the test screen, so might be nice to have a bit more info on this on a screen before going to the dates. Or maybe a dummy one.
(from OJ)

also a similar comment from Claire
